### PR TITLE
fix(highlights): give user config higher priority

### PR DESCRIPTION
**Pull request by jkunlin**

### CODE REVIEW

File: lua/avante/highlights.lua [hunk 1]
```diff
@@ -67,11 +67,11 @@ M.conflict_highlights = function(opts)
   local get_default_colors = function(key, hl)
     --- We will first check for the hl name. If plugins set this then we won't use the default opts[var_accesor]
```
Comment (yetone) [inline lua/avante/highlights.lua:68]: This comment should also be changed cc @aarnphm

Comment (aarnphm) [reply lua/avante/highlights.lua:68]: The reason I do this is that for colorscheme the can set the specific highlight group, for example the following is being used.

Then we fall back to user highlight group.

**Review by aarnphm — CHANGES_REQUESTED** [review]
refer to comment

File: lua/avante/highlights.lua (hunk 1 above)
Comment (aarnphm) [inline lua/avante/highlights.lua:68]:
```suggestion
  local get_default_colors = function(key, hl)
    --- We will first check for user custom highlight. Then fallback to default name highlight.
```

**Review by aarnphm — APPROVED** [review]

**Pull request merged**